indico/modules/events/contributions/models/contributions.py

```diff
@@ -320,6 +333,12 @@ def get_non_inheriting_objects(self):
         """Get a set of child objects that do not inherit protection"""
         return get_non_inheriting_objects(self)
 
+    def get_field_value(self, field_id):
+        field_values = {v.contribution_field_id: v.value for v in self.field_values}
+        if field_id in field_values:
```

Comment:
Why build a dict just to get a single element from it?

    return next((v.value for v in self.field_values if v.contribution_field_id == field_id), None)